easybuild/easyconfigs/r/R/R-4.2.1-foss-2022a.eb

```
@@ -0,0 +1,3359 @@
+name = 'R'
+version = '4.2.1'
+
+homepage = 'https://www.r-project.org/'
+description = """R is a free software environment for statistical computing
+ and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['https://cloud.r-project.org/src/base/R-%(version_major)s']
+sources = [SOURCE_TAR_GZ]
+checksums = ['4d52db486d27848e54613d4ee977ad952ec08ce17807e1b525b10cd4436c643f']
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('Xvfb', '21.1.3'),
+]
```

required for (at least) `httpuv`
```suggestion
    ('Autotools', '20220317'),
]
```